src/models/tools/geometry/jxg-board.ts

```diff
@@ -80,15 +80,14 @@ function getCanvasScale(eltOrId: string | HTMLElement | null) {
 
 function scaleBoundingBoxToElement(domElementID: string, changeProps: any) {
   const elt = document.getElementById(domElementID);
-  let eltBounds = elt && elt.getBoundingClientRect();
-  if (!eltBounds || !(eltBounds.width > 0) || !(eltBounds.height > 0)) {
-    eltBounds = { width: kGeometryDefaultWidth, height: kGeometryDefaultHeight } as ClientRect;
-  }
+  const eltBounds = elt?.getBoundingClientRect();
+  const eltWidth = eltBounds?.width || kGeometryDefaultWidth;
```

Comment:
Should this use "nullish coalescing" instead in case the width or height is zero?

```
const eltWidth = eltBounds?.width ?? kGeometryDefaultWidth;
const eltHeight = eltBounds?.height ?? kGeometryDefaultHeight;
```

http://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
I considered that, but I want to use the default in case of 0 as well.